docs/source/notebooks/DEMetropolisZ_EfficiencyComparison.ipynb

```diff
@@ -0,0 +1,2066 @@
fTbO+NbOHtJP7j9zOcOzAUNIntlwovzkAAAAAmE09tngh7fjxbj+bZlX/4oW0/Wu7/WyIwBcAAACA1ageT0YPz9HKoMPetGNHuv1sWlQTQesMweu0sHa2kHZ9UlXdfkIsIoEvAAAAACtDXSdjRzsMaeeqqD2UpO72M2rWv3GBIW3LmIGNSdXX7WfDCiXwBQAAAODUjB1fpJB2pLRPWEn61s0cvM4rpN2kLy3Lwv9lAAAAAKej8ZMTQesi9KYdP9HtZ9OsGpi9lUHHvWmHykJk0EN6NvCtquqMJL+U5IlJdibZmuShJLck+X+TfKyu647q9quqWpvk9UlenOSiJAeS/EWSN9Z1vW/xZw9LYPjmZOxwcsbj9d4BAABYreqx0qpgtn6znfamHTva7WfTolqkkHZz0rfWe2NOWz0b+CY5M8nLk3whyR8n2Z/krCQ/nOSjSW5I8m/mOklVVX1J/iTJs5N8McnHklya5GVJnlVV1ZOFvqx43/1i8r+fmqRONu1Kdr4i2fHSZN3Z3Z4ZAAAAdV0W/ZqtSrbTtgejh7r9bKYb2DT7omCdLiDWv0FIC4uglwPfO5KcUdf1aOOdVVUNpYTAr6yq6l11Xf/THOd5aUrY+3tJXjRZFVxV1cuSvD/J2yeOgZXrvj/LI43oD92a3PT65BtvSM57bgl/z/nBpOrv6hQBAAB6Sl0n48cXJ6Q9OZJkvNvPqFn/+sUJaQc2WTwMVpieDXzrun0H77quR6qq+qsku5M8Oslcge8rJ27/Y2MLiLquf6uqqtcm+Ymqql5V1/XIYswblsTwN6ffV48m936sbBsuSna8PNn5srIPAACwWo2fmL3dwXx60zbXmHVfNZgMblmcBcQsHgar1qr7011V1bok359S7ring2OfnOSWuq73tjnkr5P8uyRPSfLJOc71+RkeumquOcMpawx8n/D+5L6/SL7zJ1MvTo7clex5S7Lnrck5zylVv+f9cNI32JXpAgAANKnHTjGkbRgzfrzbz6ZZ1T978Dqf3rT9a7v9bIAe0POB78Tiba9J0pfSw/dfJrkwyVvrur51juGXToyb6bjJ+3dljsAXumb8ZGnjkJR+RzteWip5jz2Q7P2d5Pb3Joe+NXFwnez7eNnWnpXsuL6Ev0O7ujV7AACgV9Xjyejh2RcF6zSkHTvS7WfToiqtChZjAbH+9frSAsuq5wPfJGckeXPDzyeTvDbJOzoYu2Xi9uAMjw+3HDejuq6f2u7+icrfp3QwF1iYQ7dNVfIOXTHVO2ndWcnlr00u+8Xkoc8mt9+Q3PPRZPxYefz4A8ktv1q27U9Pdr4yueD5Sf+67jwPAABg6dV1MnZ07lYGHfWmPZRH1hJZKfo3LE5IO7BRX1qgZ/V84FvX9Z1Jqqqq+lMqe38yya8k+Z6qqn68dVE3WHUa2zlsvnL641WVbL+ubNe+O7nrQyX8PXjj1DEP/m3Zvvrq5OIXJ5e8Mtly9dLPHQAA6MzY8blD2k5707ZfEqd7+tbOsShYhwuIDWzSlxYgqyDwnTSxiNudSd5WVdVYkl9NWZDtN2cZNlnZO1MF7+aW42DlGWkMfHfPfuyarcmjX5Vc+nPJga+U4PfuD098Mp/k5IHktveUbduTStXvRT9ZXjgBAADzMz66CCHtxO34iW4/m2bVwNyLgnW0gNhQ0rem288GYFVZNYFvi79OCXyfkdkD328nGU/p0dvO5P1z9QKG7hluWJuwXYVvO1WVbHti2R73zuTuj5Rev/u/MHXM/i+V7eu/kFz4k6Xqd+sT9Z4CAGB1q8dLQcRs/WY7DWnHjnb72bSoFimk3Vyqcr03AFiRVmvge97E7aztHOq6PlZV1ZeSPKWqqovrut7bcsgPJjme5ItLMEdYHI0tHYY6DHwbDWxKdv502Q7eVILfvR8o1b5JebF7x3vLtuXqUvV78U8la7YtzvwBAOBU1XVZ9GuuVgad9Kad/PbbSjKwafZWBp32pu3fIKQFOA30bOBbVdXjktxR1/XBlvu3JflPEz9+vOH+c1NaN9zXMuZ/pCyq9raqql5U13U9cfzLklyZ5Hfruh4OrET1eDJyc9mvBpNNl57a+bZcnVz7ruSatyf3fqyEvw9+aurxgzclX/v55MbXJhe8oFT9nnmdF40AAMxfXSfjxxcnpD05kvLlzRWkf/3cIW1HvWk3JlV/t58NAD2kZwPfJNcneUVVVZ9KsjfJ4SQXJ/lXSTYl+cMk/7Ph+P+c5KVJXpbktxvu/90kP5Gy2NvOqqo+neSSJM9PcneS1y/hc4BTc+Suqa+JDe1K+gYX57z965KLXlS2kVuTO96X3PnbyfH7y+Pjx8vib3d9KNm0K9n5imTHS5N1Zy/O9QEAWLnGT3YW0naygNhKW2O7GmwIYE8lpN20eK/NAWCeejnw/WhKxe5TklyXZEOS/Un+LiXE/b3Jat3Z1HU9VlXVj6QEuy9J8gtJDqSEwm+s63rfksweFkNj/96FtHPoxNCu5Jq3JVf9cnLfn5eq332fyCMVFIduTW56ffKNNyTnPbdU/Z79LFUIAAArST02EbTO0m+2096048e6/WyaVf2LUEk7cdu/ttvPBgBOWc8GvnVd/11KuNvp8denVAW3e+x4kl+a2KB3NPbv7XTBtoXqG0zO/7GyHbk7ufO3SuXvkbvK4/VoaQNx78eSDRclO16e7HxZ2QcAYP7q8WT08AJD2pYxY4e7/WxaVKUKdjEWEOtfr8UYADTo2cAXSDLSGPjuXr7rbrgw2f2m5Mo3JPf/r+T2G5Lv/MnUV/KO3JXseUuy563JOc8pLR/O+2FfawMAVr+6Li23Zu0322lv2pEkc35pcXn1b1hASNsmrB3YmFR93X42ALAqCXyhlzW2dFjqCt92qv7knGeX7dj9yd7fLS0fDn1r4oA62ffxsq09K9lxfQl/h3Yt/1wBAGYzdrzzkHautgf1WLefTbO+tXP0m+00pN2U9HkLCQArnX+toVfVdUNLhyoZuryr08m
```

Comment:
Shouldn't be "DEMetropolisZ can achieve the same effective sample sizes with less chains."